Misc/NEWS.d/next/Library/2019-07-27-18-00-43.bpo-37689.glEmZi.rst

```diff
@@ -0,0 +1 @@
+Add an is_relative_to() in PurePath class to compute a path is relative to the path or not.
```

Comment:
Add rst inline markup and adjust the phrasing:
```suggestion
Add :meth:`is_relative_to` in :class:`PurePath` to determine whether or not one path is relative to another.
```

Reply:
done